packages/backend/src/schema.mjs

```diff
@@ -38,5 +51,5 @@ export default [
       ['data', 'String'],
     ],
   },
-  ...schema,
+  ..._schema,
```

Comment:
@vimwitch  did I do this right? New error is
> Error: anondb error: Error: SQLITE_ERROR: near ")": syntax error
    at /Users/Colleen/Documents/UniRep/explorer/node_modules/anondb/connectors/sqlite.js:235:23
    at async Synchronizer.processEvents (/Users/Colleen/Documents/UniRep/explorer/node_modules/@unirep/core/src/Synchronizer.js:309:17)
    at async Synchronizer.poll (/Users/Colleen/Documents/UniRep/explorer/node_modules/@unirep/core/src/Synchronizer.js:254:9)
    at async /Users/Colleen/Documents/UniRep/explorer/node_modules/@unirep/core/src/Synchronizer.js:193:42